lib/graphql/document.ts

```diff
@@ -0,0 +1,68 @@
+import { GraphQLClient } from "./client";
+
+import {
+  Document as DocumentModel,
+  MutationUpdateDocumentArgs,
+} from "./schema";
+
+const DOCUMENT_FIELDS = `
+id
+name
+type
+url
+note
+createdAt
+`;
+
+const FETCH_DOCUMENTS = `
+  query {
+    viewer {
+      documents {
+        ${DOCUMENT_FIELDS}
```

Comment:
I wonder sometimes, if we hardcode the response values like this in sdk, isn't it destroying a bit a benefit of gql:
Having flexibility and beeing able to decide on the client what particular fields we want to have in the response.

Reply:
Updated with https://github.com/kontist/js-sdk/pull/252/commits/8a7b7559e645b0339901ffb45cf4c4c34587c2d9